deps.edn

```diff
@@ -901,15 +901,17 @@
   ;; https://github.com/borkdude/clj-kondo/
   :lint/clj-kondo
   {:extra-deps {clj-kondo/clj-kondo {:mvn/version "2021.12.16"}}
-   :main-opts  ["-m" "clj-kondo.main" "--lint" "src"]}
+   :main-opts  ["-m" "clj-kondo.main" "--lint" "src" "test"]}
 
 
   ;; Eastwood - classic lint tool
```

Comment:
To be more specific, perhaps this should be 
Eastwood - lint too using clojure/tools.analyzer.jvm